document/CoreFormat.mdk

```diff
@@ -355,12 +359,52 @@ An `issue` object describes a single issue detected by a static analysis tool.
 ### `ruleId` property { #issue-ruleId }
 
 An `issue` object shall contain a property named `ruleId` whose value is a string containing
-a stable, opaque identifier for the rule that was violated.
+the stable, opaque identifier for the rule that was violated.
 
 ~ IsoExample
 `"CA2101"`
 ~
 
+### `kind` property { #issue-kind }
+
+An `issue` object may contain a property named `kind` whose value is one of a fixed set of strings
+that specify what kind of issue this is.
+
+If present, the `kind` property shall have one of the following values, with the specified meanings:
+
+* `warning`: The rule specified by the `ruleId` property (&sect;[#issue-ruleId]) was evaluated and a violation was found.
+
+* `error`: The rule specified by the `ruleId` property was evaluated and a serious violation was found.
+
+* `pending`: The rule specified by the `ruleId` property was evaluated, but whether the result of the evaluation
+constitutes a violation cannot be determined without further evaluation, either by a person or by an automated tool.
+
+~ IsoNote
+In some cases, team policy or corporate policy might determine if an analysis result constitutes a violation.
```

Comment:
Here's an example. A binary checker might observe that an analysis target was compiled with a tool chain of a specific version. Subsequently, a post-processing tool might examine this result in order to determine whether that version is within policy. This approach enables the ability to decouple a standard of enforcement from salient facts relevant to assessment. 

btw - for the record, we have not identified a good pattern for extracting and evaluating 'pending' messages in a post-processing step. doing so might require updates to this standard. we could therefore consider dropping 'pending' from this version while someone actually builds a practical application of this idea.